### DISCUSSION TITLE
input and output of pathfindR

### DISCUSSION THREAD

**Discussion by ManarRashad** [egeulgen/pathfindR#52] [Q&A] [answered]
Dear all,
I just need to ask about the input of pathfindR:
I know that the input is table contains features, logfoldchange(optional) 
my question : the third column contains p value or adjusted p value ?

Second Question: the output contains kegg pathways with enrichment score and p adjusted (lowest and highest)
while make 10 iterations, 
1- if I want to find the p value, can I multiply the resulted p adjusted by the number of iterations as the iteration equal the number of tests or there are some ways to find p value?
2- if I need to find 1 p-adjusted value for the kegg pathway to be used for further analysis, can I get the average of the lowest and highest resulted p adjusted value or there is a way to find 1 p-adjusted ?

**Comment by egeulgen** [comment, marked as answer]
Dear @ManarRashad,

- We would suggest using adjusted p-values
- the p values reported are the lowest and the highest p values obtained over all iterations. To use only 1 p value, I would suggest using the lowest p-value.
